lib/nostrum/struct/component/option.ex

```diff
@@ -14,11 +14,26 @@ defmodule Nostrum.Struct.Component.Option do
     :default
   ]
 
+  @typedoc """
+  The user-facing name of the option.
+  """
   @type label :: Component.label()
+
+  @typedoc """
+  The developer defined value of the option.
+  """
   @type value :: String.t()
-  @type description :: Component.description()
+
+  @typedoc """
+  An optional description of the option. Max length is 100 characters.
+  """
+  @type description :: String.t() | nil
   @type emoji :: Component.emoji()
-  @type default :: Component.default()
+
+  @typedoc """
+  If this option is selected by default.
+  """
+  @type default :: boolean() | nil
```

Comment:
Argh, this will also conflict with master, sorry :( Feel free to overwrite it with your changes!